src/gmm/mle-diag-gmm.h

```diff
@@ -66,6 +70,14 @@ struct MleDiagGmmOptions {
                  module+"Variance floor (absolute variance).");
     opts->Register("remove-low-count-gaussians", &remove_low_count_gaussians,
                  module+"If true, remove Gaussians that fall below the floors.");
+    opts->Register("variance-floor-value", &variance_floor_value,
+                 module+"Value assigned to all elements of variance_floor_vector.");
+    opts->Register("variance-floor-vector-dim", &VFV_dim,
+                 module+"dimension of variance_floor_vector.");
+  }
+  void create_VFV() {
```

Comment:
this code doesn't have the right style (study Google style).  I suspect it's not needed anyway for the overal commit.